coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -962,7 +965,8 @@ def linear(context, node):
     context.add(res, torch_name=node.name)
 
 
-@register_torch_op(torch_alias=["conv2d", "convolution"])
+# NOTE: This function is also an alias of: ["conv_transpose1d", "conv_transpose2d", "conv_transpose3d"] but we lack tests for those
```

Comment:
Unfortunately, I fail to add test for `conv_transposeXd` because it asks for `output_size` as an input in the computational graph which I don't know how to solve. Though, the function should support the operation.